### DISCUSSION TITLE
How to deal with formula errors?

### DISCUSSION THREAD

**Discussion by Sunny19191** [web-infra-dev/rspress#1448] [Q&A]
When build the doc, there are several cases where the formula reports an error as below:
1. Error: "15:24: Could not parse expression with swc: Unexpected content after expression"
2. Error: "15:37: Could not parse expression with swc: Expected unicode escape"
3. Error: "41:104: Could not parse expression with swc: Identifier cannot follow number"
Could you provide some suggestions about it?

**Comment by Timeless0911** [comment]
Please take a minimal reproduce demo.

**Reply by Sunny19191** [reply]
Examples:
1.$$dst(x,y)=src(\frac{M_{11} x+M_{12} y+M_{13}}{M_{31}x+M_{32}y+M_{33}},\frac{M_{21}x+M_{22}y+M_{23}}{M_{31}x+M_{32}y+M_{33}})$$
2.$${dst} = \Delta \text{src} = \frac{\partial^2 \text{src}}{\partial x^2} + \frac{\partial^2 \text{src}}{\partial y^2}$$
3.$$F(u,v)=\cfrac{1}{4}C(u)C(v)[\displaystyle\sum_{m=0}^{7} \displaystyle\sum_{n=0}^{7}f(m,n)cos\cfrac{(2m+1)u\pi}{16}cos\cfrac{(2n+1)v\pi}{16}]$$

**Reply by Timeless0911** [reply]
You should use code block with [rspress-plugin-katex](https://github.com/linbudu599/rspress-plugins/tree/main/packages/rspress-plugin-katex).

**Reply by Sunny19191** [reply]
The katex formula is used, and the doc can be built successful and the formula is displayed right. But during the compilation process, the log will be appear some errors about the formula, I want to reslove these log errors about the formula as belows.
![image](https://github.com/user-attachments/assets/35bcf3be-9c59-4c5a-a356-0f31fc815628)

**Reply by Timeless0911** [reply]
I can not reproduce, can you provide your minimal repo demo?

**Reply by Sunny19191** [reply]
You can use the examples provided above to compile it directly，then the log will be appeared some errors about the formula.

**Reply by Timeless0911** [reply]
![image](https://github.com/user-attachments/assets/42531836-dbb0-4f3c-adbc-dd20fcf814bb)
nothing
